SECURITY-INSIGHTS.yml

```diff
@@ -0,0 +1,63 @@
+header:
+  schema-version: 1.0.0
+  expiration-date: '2023-08-31T10:10:09.000Z'
+  last-updated: '2021-09-01'
+  last-reviewed: '2022-09-01'
```

Comment:
should this be the day of the release?

Reply:
It should

Reply:
also Kueue started  on 22, so 21 is incorrect, please fix

Reply:
Updated in the latest commit